hack/operatorhub/config.yaml

```diff
@@ -30,6 +30,9 @@ crds:
   - name: stackconfigpolicies.stackconfigpolicy.k8s.elastic.co
     displayName: Elastic Stack Config Policy
     description: Elastic Stack Config Policy
+  - name: logstashes.logstash.k8s.elastic.co
```

Comment:
We may also want to update [hack/operatorhub/templates/csv.tpl](https://github.com/elastic/cloud-on-k8s/blob/main/hack/operatorhub/templates/csv.tpl)

Reply:
Done